com.github.chipmuenk.pyfda.yaml

```
@@ -0,0 +1,58 @@
+app-id: com.github.chipmuenk.pyFDA
+runtime: org.kde.Platform
+runtime-version: '5.12'
```

this runtime is EOL.
```suggestion
runtime-version: '5.15-21.08'
```